### ISSUE TITLE
failed to install on Openpli 8.3 and 9.0

### ISSUE THREAD

**Issue by isthecloud** [m4dhouse/Wireguard-Vpn#12]
Hi,

I am on Openpli 8.3 trying to install the .ipk ; I did the following:

>cd /tmp
>wget enigma2-plugin-extensions-wireguard-vpn_9.4_all.ipk
>opkg install enigma2-plugin-extensions-wireguard-vpn_9.4_all.ipk
Collected errors:
 * open_outer: Failed to open package 'enigma2-plugin-extensions-wireguard-vpn_9.4_all.ipk': Unrecognized archive format
 * open_outer: Failed to open package 'enigma2-plugin-extensions-wireguard-vpn_9.4_all.ipk': Unrecognized archive format
 * pkg_extract_control_file_to_stream: Failed to extract control.tar.* from package 'enigma2-plugin-extensions-wireguard-vpn_9.4_all.ipk'.
 * pkg_init_from_file: Failed to extract control file from enigma2-plugin-extensions-wireguard-vpn_9.4_all.ipk.

I did try to install in on an Openpli 9.0 with no luck either.

Update: I just saw a closed issue and try to install https://github.com/m4dhouse/Wireguard-Vpn/tree/python-3.9 on the openpli 8.3 but i had same problem. 

Any help will much much appreciated. 

Thanks

**Comment by m4dhouse** [comment]
Please refer to the python version of your image based on the chosen branch.
Branch "python3" = python 3.11.x
Try the latest version 9.6

**Comment by isthecloud** [comment]
Thanks for your reply.

I have tried all your 3 branches with no luck. 

I tried the 3 branches in both OpenPLi 8.3 and 9.0 ; the 9.0 from what I understood is working on phyton 3.0 but the 8.3 I think it is older version of phyton.

The result is this:

root@gbue4k:/tmp# opkg install enigma2-plugin-extensions-wireguard-vpn_9.6_all.ipk
 * open_outer: Failed to open package 'enigma2-plugin-extensions-wireguard-vpn_9.6_all.ipk': Unrecognized archive format
 * open_outer: Failed to open package 'enigma2-plugin-extensions-wireguard-vpn_9.6_all.ipk': Unrecognized archive format
 * pkg_extract_control_file_to_stream: Failed to extract control.tar.* from package 'enigma2-plugin-extensions-wireguard-vpn_9.6_all.ipk'.
 * pkg_init_from_file: Failed to extract control file from enigma2-plugin-extensions-wireguard-vpn_9.6_all.ipk.
